src/pykeen/experiments/literale/kristiadi2019_complex+literale_glin_fb15k237.yaml

```diff
@@ -0,0 +1,61 @@
+metadata:
+  title: "Learn FB15k-237 Dataset with ComplEx+LiteralE-g_lin as described by Kristiadi et al., 2019"
+pipeline:
+  dataset: FB15k237WithLiterals
+  dataset_kwargs:
+    create_inverse_triples: true
+    numeric_literals_preprocessing: "minmax"
+    numeric_triples_preprocessing: "filter_by_relations"
+    numeric_triples_preprocessing_kwargs:
+      min_occurrences: 5 # https://github.com/SmartDataAnalytics/LiteralE/blob/0b0c48fd9b74bf000400199610275ea5c159a44c/filter_literal.py#L11
+      regex: '^(?!.*http://rdf\.freebase\.com/key/).*$' # https://github.com/SmartDataAnalytics/LiteralE/blob/0b0c48fd9b74bf000400199610275ea5c159a44c/filter_literal.py#L12
+  evaluator: RankBasedEvaluator
+  model: ComplExLiteral
+  model_kwargs:
+    embedding_dim: 200
+    input_dropout: 0.2
+    loss: BCEWithLogitsLoss
+  epochs: 100
+  optimizer_kwargs:
+    lr: 0.001
+  training_loop: LCWA
+  training_kwargs:
+    batch_size: 128
+    label_smoothing: 0.1
+    num_workers: 4 # https://github.com/TimDettmers/spodernet/blob/afa1c4cee8b168fd2938067b37ca4795a022d91f/spodernet/preprocessing/batching.py#L214
+    worker_init_fn: "literale_worker_init" # for references see literale_worker_init() function in pykeen/utils.py
+    drop_last: True # https://github.com/SmartDataAnalytics/LiteralE/blob/master/README_ConvE.md#quirks
+  stopper: "early"
+  stopper_kwargs:
+    metric: "inverse_harmonic_mean_rank"
+    frequency: 1 # added after communication with paper's author
+    patience: 3 # added after communication with paper's author
+    relative_delta: 0.000001 # = 1e-6 | added after communication with paper's author
+  random_seed: 1677423275 # https://github.com/SmartDataAnalytics/LiteralE/blob/0b0c48fd9b74bf000400199610275ea5c159a44c/main_literal.py#L43-L49
+results:
+  realistic:
```

Comment:
Are these results from your own reproduction? Or from the original publication?

Reply:
All the experiment results are from my own reproduction.